doc/tutorials/using_windows_as_uos.rst

```diff
@@ -1,8 +1,8 @@
 .. _using_windows_as_uos:
 
-Using Windows as Guest OS on ACRN
+Using Windows as Guest VM on ACRN
 #################################
-This tutorial describes how to launch Windows as the Guest OS on the ACRN hypervisor.
+This tutorial describes how to launch Windows as the Guest VM on the ACRN hypervisor.
```

Comment:
Let's add the acronym here too (and use "a" instead of "the" to match the acronym:
```
This tutorial describes how to launch Windows as a Guest (WaaG) VM on the ACRN hypervisor.
 
```

Reply:
Fixed :smile: